docs/src/TUTORIAL.md

```diff
@@ -0,0 +1,162 @@
+# Tutorial
+
+Pour comprendre le potentiel et comment utiliser la Gas Station, nous allons reprendre l'exemple simple des NFT disponible à cette adresse : https://gas-station-nft-example.marigold.dev
+
+## Template
+
+La première chose à faire est de récupérer le template de code situé [ici](https://github.com/marigold-dev/gas-station-lib).
+Une fois récupéré, vous pouvez run ces commandes :
+```
+npm install
+npm run check
+```
+Pour tester que tout se lance correctement, vous pouvez utilise
+```
+npm run dev
+```
+
+Les fichiers qui vont nous intéressés se situent dans `src/lib`. Vous allez y trouver des composants en Svelte et un fichier `tezos.ts` qui regroupe des fonctions utilitaires comme la connexion au wallet, etc...
+
+## Minting
+
+Nous allons commencer par le mint d'un NFT par un utilisateur. Le contrat que nous allons utiliser est disponible à cette adresse sur Ghostnet : `KT199yuNkHQKpy331A6fvWJtQ1uan9uya2jx`
```

Comment:
Tutorial would include a part about Permit from smart contract point of view. Developpers who will use SDK will not know about.

Minimal here would be to have an anatomy of the smart contract and make devs deploy there own. You can start with the global one and introduce this later, but i think it must be part of the tuto